pkg/admission/validator/shoot.go

```diff
@@ -22,10 +22,6 @@ import (
 
 	api "github.com/gardener/gardener-extension-provider-aws/pkg/apis/aws"
 	awsvalidation "github.com/gardener/gardener-extension-provider-aws/pkg/apis/aws/validation"
-	"github.com/gardener/gardener-extension-provider-aws/pkg/aws"
```

Comment:
Nit: Can you keep the `github.com/gardener/gardener-extension-provider-aws` imports in a separate import group as before?